packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadIncludedChanges.st

```diff
@@ -0,0 +1,6 @@
+button-actions
+loadIncludedChanges
+
+	(self includedChangeListItems
+		sorted: [:a :b | a timeStamp < b timeStamp])
```

Comment:
This PR didn't change the functionality of this one, but I just noticed that if the changes are displayed correctly, this still loads older changes in the same minute first. We should probably also turn this into `reversed` instead of using `sorted:`.
(Just a tangential thought – if you think this is okay for now, that's fine too. Could be addressed in another PR.)

Reply:
We will skip this sorting at all, since the Items should be sorted anyways